x-pack/plugins/apm/public/components/app/diagnostics/index_pattern_settings_tab.tsx

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiLink, EuiLoadingElastic } from '@elastic/eui';
+import {
+  EuiBadge,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+  EuiToolTip,
+} from '@elastic/eui';
+import React from 'react';
+import { APIReturnType } from '../../../services/rest/create_call_apm_api';
+import { useApmRouter } from '../../../hooks/use_apm_router';
+import { FETCH_STATUS } from '../../../hooks/use_fetcher';
+import { useDiagnosticsContext } from './context/use_diagnostics';
+
+type DiagnosticsBundle = APIReturnType<'GET /internal/apm/diagnostics'>;
```

Comment:
Would it be useful just to export this type under `app/diagnostics` folder? I've seen this line in multiple files